apps/web/tsconfig.json

```diff
@@ -8,9 +8,9 @@
   ],
   "compilerOptions": {
     "checkJs": true,
-    "verbatimModuleSyntax": true,
     "skipLibCheck": true,
     "strict": true,
-    "noEmit": true
+    "noEmit": true,
+    "types": ["vitest/globals"]
```

Comment:
I think we should only add vitest globals to the node tsconfig, as that's the environment in which it runs